web_src/js/components/RepoActionView.vue

```diff
@@ -366,26 +375,24 @@ export function ansiLogToHTML(line) {
   margin: 0 20px 20px 20px;
 }
 
-.action-view-header .run_cancel {
+.action-view-header .action-control-button {
   border: none;
-  color: var(--color-red);
   background-color: transparent;
   outline: none;
   cursor: pointer;
   transition: transform 0.2s;
+  display: flex;
 }
 
-.action-view-header .run_approve {
-  border: none;
+.action-view-header .green {
   color: var(--color-green);
-  background-color: transparent;
-  outline: none;
-  cursor: pointer;
-  transition: transform 0.2s;
 }
 
-.action-view-header .run_cancel:hover,
-.action-view-header .run_approve:hover {
+.action-view-header .red {
+  color: var(--color-red);
```

Comment:
There is already global colors, do `text red` and `text green` work?

Reply:
Thank you very much.
`text red` and `text green` works well.